meta-resin-common/recipes-support/resin-mounts/resin-mounts.bb

```diff
@@ -84,5 +90,9 @@ do_install () {
 
         ln -sf ${systemd_unitdir}/system/etc-resin\\x2dsupervisor.mount ${D}${sysconfdir}/systemd/system/resin-bind.target.wants
         ln -sf ${systemd_unitdir}/system/etc-NetworkManager-system\\x2dconnections.mount ${D}${sysconfdir}/systemd/system/resin-bind.target.wants
+
+        install -c -m 0644 ${WORKDIR}/systemd-timesyncd.conf ${D}${sysconfdir}/systemd/system/systemd-timesyncd.service.d
+        # Also add to resin-bind wants so resin-state-reset will populate the file
+        ln -sf ${systemd_unitdir}/system/etc-systemd-timesyncd.conf.mount ${D}${sysconfdir}/systemd/system/resin-bind.target.wants
```

Comment:
This is not needed as it already has `WantedBy=resin-bind.service` which when enabled will create the required symlink.